spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+from cerberus.validator import (
+    Validator as cerberus_validator,
+    DocumentError,
+    SchemaError,
+)
+from spidermon.contrib.validation.validator import Validator
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator, use_default_translator=use_default_translator
+        )
+        self._schema = schema
+
+    def _validate(self, data, strict=False):
+        try:
+            validator = cerberus_validator(self._schema)
+            validator.validate(data)
+            for field_name, error in validator.errors.items():
+                self._add_errors({field_name: error})
+        except DocumentError:
```

Comment:
It is better to raise an exception if the schema is not provided as soon as possible. We shouldn't wait until calling `_validate`. I'd prefer this validation in `__init__`.

Reply:
Completely agree with that. Should I work towardsraise a new error there or shift this only.

Here's something I have in mind. Borrowed from Cerberus Source. 

```
 if schema is not None:
            self._schema = schema
        elif self._schema is None:
            if isinstance(self._schema, Mapping):
                self._schema = {}
            else:
                raise SchemaError("validation schema missing")
```